_includes/master/reqs-sys.md

```diff
@@ -30,10 +30,15 @@ to connect to, we provide instructions in the [installation documentation](./ins
 
 ## Network requirements
 
+{{site.prodname}} requires the network to allow the following types of traffic.
+
 | Traffic | Protocol | Port |
 | ------- | -------- | ---- |
 | BGP     | TCP      | 179  |
-| IPIP    | 4        | n/a  |
+| IPIP\*  | 4        | n/a  |
+
+\* Our manifests enabled IPIP by default. If you disable IPIP, you won't need to
```

Comment:
typo: enabled

Minor nit, but this makes it sound like you can just disable IPIP and not have to do this, but there are considerations that need to be made before disabling IPIP otherwise it won't work in some environments.

Maybe link here for more details? https://docs.projectcalico.org/v3.0/usage/configuration/ip-in-ip